packages/web3-errors/src/errors/transaction_errors.ts

```diff
@@ -84,16 +85,21 @@ export class RevertInstructionError extends BaseWeb3Error {
 	}
 }
 
-export class TransactionRevertError extends BaseWeb3Error {
+export class TransactionRevertInstructionError<
```

Comment:
Renaming to keep parity with [1.x](https://github.com/web3/web3.js/blob/8d369a9e76267df9ce666283521c1fd6d3f77fc4/packages/web3-core-helpers/src/errors.js#L87)